src/reactapp/src/hook/useFormikMemorizer.js

```diff
@@ -12,6 +12,8 @@ export default function useFormikMemorizer(formSectionId) {
     setFieldTouched,
   } = useFormikContext();
 
+  // console.log({ errors, values })
```

Comment:
can be removed

Reply:
Okay, there is another performance improvement PR coming. this will be removed there.